crates/turborepo-lib/src/run/summary/mod.rs

```diff
@@ -347,13 +347,13 @@ struct SinglePackageRunSummary<'a> {
 
 impl<'a> From<&'a RunSummary<'a>> for SinglePackageRunSummary<'a> {
     fn from(run_summary: &'a RunSummary<'a>) -> Self {
-        let mut tasks = run_summary
+        let tasks = run_summary
             .tasks
             .iter()
             .cloned()
             .map(SinglePackageTaskSummary::from)
             .collect::<Vec<_>>();
-        tasks.sort_by(|t1, t2| t1.task_id.cmp(&t2.task_id));
```

Comment:
Removing extra `sort()` because tasks are sorted during normalization: https://github.com/vercel/turbo/blob/1dd2e93e84e08872f67943804a6600f0e91a124a/crates/turborepo-lib/src/run/summary/mod.rs#L770

Since this sort only applies to `SinglePackageRunSummary`, it made more sense to do it when we care about sorting. If the implementation is the same as in Go, it _should_ apply to dry runs as well as run summary!

I can also revert this part if we aren't confident about that.

Reply:
I would say we remove the second sort in `normalize` over this one as other callers might depend on the list being sorted.

Reply:
But then I'd also need another sort somewhere else for monorepos (this one that is removed is only for when converting from monorepo to single package summary). 

but point made, I'll keep this one and the duplicate since it's not related to this fix anyway